### DISCUSSION TITLE
how to restart the ngrok agent

### DISCUSSION THREAD

**Discussion by globulerougee** [ddev/ddev#3536] [Q&A]
Please help me !
This is the message. I don't know what to do.

Welcome to ngrok!

Before you can serve HTML content, you must sign up for an ngrok account and install your authtoken.
I already have an account, why am I seeing this page?

Make sure that you Install Your Authtoken and then restart the ngrok agent.

**Comment by rfay** [comment]
You are seeing that message because you have not yet installed the authtoken. 

You can look in ~/.ngrok2/ngrok.yml to see what is current installed.

The ngrok setup instructions are at https://dashboard.ngrok.com/get-started/setup

It says to download ngrok and use it in current directory with `./ngrok authtoken <token>` but most people have ngrok installed in their path so it makes more sense to just `ngrok authtoken <token>`. Alternately, you can edit ~/.ngrok2/ngrok.yml

As far as "restarting the ngrok agent", that's just `ddev share`

**Reply by fellixix** [reply]
pls can you send me short video tutorial of how you did it maybe im not getting something here is my whatsapp number +14028921154

**Reply by fellixix** [reply]
i'm still facing the same problem even after installing my ngrok authtoken
how do i restart the ngrok agent

**Reply by rfay** [reply]
When you execute `ddev share` it tells you what it's doing:
```
$ ddev share
Running /opt/homebrew/bin/ngrok http http://127.0.0.1:64652
```

Here it's running the ngrok command (not really an agent) with one argument, the local port of the ddev project. That's it. You can run that command manually, `ngrok http http://127.0.0.1:64652` in my case. If you then get the error you are getting, it means you don't have the ngrok token properly installed. Please do the things I said earlier.
